Mlh 10 v2

Krismalik opened the pull request.

Krismalik committed:
test

Krismalik committed:
-added data testing for story
-added 4 test cases 'Try again button'
-added files for story in repos. Smoke and Regres

MaksymLyk reviewed (changes requested):
INFO @wdio/cli:launcher: Run onPrepare hook
2020-11-20T05:37:16.288Z ERROR @wdio/cli:launcher: No specs found to run, exiting with failure
2020-11-20T05:37:16.288Z INFO @wdio/cli:launcher: Run onComplete hook
2020-11-20T05:37:16.289Z INFO @wdio/selenium-standalone-service: shutting down all browsers

Spec Files:      0 passed, 0 total (0% completed) in 00:00:01

2020-11-20T05:37:16.290Z INFO @wdio/local-runner: Shutting down spawned worker
2020-11-20T05:37:16.554Z INFO @wdio/local-runner: Waiting for 0 to shut down gracefully
2020-11-20T05:37:16.554Z INFO @wdio/local-runner: shutting down
npm ERR! Test failed.  See above for more details.

Krismalik committed:
-added data testing for story
-added 4 test cases 'Try again button'
-added files for story in repos. Smoke and Regres
